backend/hyperglosae/src/views/lib/links.js

```diff
@@ -1,36 +1,55 @@
 exports.parseReference = parseReference = (reference) => {
-  let [id, fragment] = reference.split('#');
-  return {id, fragment};
-}
+  let [id, fragment] = reference.split("#");
+  return { id, fragment };
+};
 
 // Note: The result set includes also the document itself or its main document
-exports.getRelatedDocuments = ({isPartOf, links}) =>
+exports.getRelatedDocuments = ({ isPartOf, links }) =>
```

Comment:
This line is not related to your contribution please don't include it in your commits.